en/config/actuators.md

```diff
@@ -1,139 +1,483 @@
+   - If nothing moves then increase the slider mid-way though the range, then higher if needed. 
+     If nothing moves after that the output might not be connected, the motor might not be powered, or the output might be misconfigured.
+	 You will need to troubleshoot (perhaps try other actuators to see if "anything" moves.
+1. Return the slider to the minimal position.
+1. Repeat for all motors
+
+Actuator assignment can be checked in a similar way.
+
+
+### Motor Configuration
+
+The motor configuaration sets output values such that motors:
+- don't spin when disarmed (at the `disarmed` PWM output value)
+- barely spin at the `minimum` PWM output value
+- give **positive thrust** in the expected direction
+
+For each motor:
+
+1. Pull the motor slider down so that it snaps to the bottom.
+   In this position the motor is set to the outputs `Disarmed` value.
+   - Verify that the motor doesn't spin in this position.
+   - If the motor spins, reduce the corresponding PWM `Disarmed` value in the [Actuator Outputs](#actuator-outputs) section to below the level at which it still spins.
+2. Slowly move the slider up until it snaps to the _minimum_ position.
+   In this position the motor is set to the outputs `Minimum` value.  
+   - Verify that the motor is spinning very slowly in this position.
+   - If the motor is not spinning, or spinning too fast you will need to adjust the corresponding PWM `Disarmed` value in the [Actuator Outputs](#actuator-outputs) such that the motors barely spin.
+   
+     ![PWM Minimum Output](../../assets/config/actuators/pwm_minimum_output.png)
+     :::note
+     For DShot output, this is not required <!-- any, or just the minimum check? -->
+     :::
+3. Increase the slider value to a level where you can verify that the motor is spinning in the correct direction and that it would give a positive thrust in the expected direction.
+   - The expected thrust direction can vary by vehicle type, for example in multicopters the thrust should always point upwards, while in a fixed wing vehicle the thrust will push the vehicle forwards.
+   - For VTOL, thrust should point upwards when the Tilt Servo is at 0 degrees as defined the [Tilt Servo Convention](#motor-tilt-servos-conceptual).
+     Testing of the [Tilt Servo](#tilt-servo-testing) is covered below as well.
+   - If thrust is in the wrong direction, you may need to [reverse the motors](#reversing-motors).
+   
+
+### Control Surface Setup
+
+The control surface configuaration sets output values such that the surfaces:
+
+1. have the correct idle position when disarmed.
+1. move in the directions as defined in the [Control Surface Convention](#control-surface-direction-conceptual).
+
+![Control Surface Disarmed 1500 Setting](../../assets/config/actuators/control_surface_disarmed_1500.png)
+
+For each control surface:
+1. Set the `Disarmed` value for Control Surface outputs to `1500` (note, this assumes we're using a PWM actuator where 1500 is the middle of the range)
```

Comment:
@junwoo091400 I gave it a big edit of the output section. Quite happy with the structure now - following your idea that they are entwined with testing. Also separated out an output assignment section that is "manual".

Just this section and the one after to go. My problem is that, while I get the point, I don't think it is generic.

For a start, are you saying that all control surfaces are controlled using PWM values of range 1000 to 2000. Reading this I think so. Further, I think that you set the value always to 1500. Is that correct, or would you adjust it if it was in the wrong position? Or would you adjust the trim value in the geometry?

Reply:
The 'Disarmed' value and 'Trim' values are decoupled and can be quite confusing.

'Disarmed' is which command the Servo will be commanded when the vehicle is disarmed. And it's absolute value / precision is not so relevant, as you just want to have the control surfaces in a reasonable position (and not in min/max position).

'Trim' is relevant for when the control surface is actively being controlled. As far as I know, if the trim = 0, command will be based on the average of the minimum / maximum PWM range. So the 'disarmed' isn't related at all.